vm.js

```diff
@@ -3273,318 +3662,23 @@ Object.subclass('Squeak.Primitives',
         }
         return false;
     },
+
+    getPrimitiveFunc: function(primIndex, argCount, primMethod) {
+        if ((primIndex > 264) && (primIndex < 520)) {
+            return this.primitiveFunctions[260];
```

Comment:
I already thought some time ago that this is never called, but forgot about it. Most likely not as tryPrimitive handels those directly.